python/evaluation_tools/run_experiment.py

```diff
@@ -84,12 +84,36 @@ def __init__(self,
           + '_' + self.eval_dict["experiment_name"]
 
     # Find sensors file:
-    sensors_file = ''
+    sensors_filepath = ''
     if 'sensors_file' in self.eval_dict.keys():
-      sensors_file = eval_utils.findFileOrDir(self.root_folder, "calibrations",
-                                              self.eval_dict['sensors_file'])
-    self.eval_dict['sensors_file'] = sensors_file
-    self.logger.info("Using sensors file: " + sensors_file)
+      sensors_file = self.eval_dict['sensors_file']
+      if type(sensors_file) == str:
+        sensors_filepath = sensors_file
+      elif type(sensors_file) == list:
```

Comment:
is this list type so that we can in the future also support different calibrations for each dataset?